extensions/hupu/README.md

```diff
@@ -0,0 +1,7 @@
+# Raycast Search Hupu
+
+通过Raycast来浏览[虎扑步行街](https://bbs.hupu.com/all-gambia)!
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

Reply:
done